### DISCUSSION TITLE
Erro ao tentar cadastrar usuário no Ubuntu: "Não há suporte para a operação tentada para o tipo de objeto a que é feita referência."

### DISCUSSION THREAD

**Discussion by ThierryS99** [balta-io/forum#105] [Tira Dúvidas] [answered]
Olá, pessoal. Tudo bem? Espero que sim!

Estou com problemas ao instalar o Ubuntu. Segui todos os passos do artigo "https://balta.io/blog/wsl". Porém ao inicializar pela primeira vez o Ubuntu aparece determinado erro: Não há suporte para a operação tentada para o tipo de objeto a que é feita referência. Quando tento inserir um novo usuario, ao clicar Enter, novamente aparece a mensagem. Se fecho o Ubuntu e abro novamente não aparece a opção de cadastrar um novo usuário, aparecendo apenas a mensagem de erro (Conforme imagem abaixo). Utilizo Windows 10 Pro, Versão 22H2 , Build 19045.2251. Agradeço no que puderem ajudar pois na internet não achei nada.

Ps.: Já instalei três versões diferentes do Ubuntu, também ja reiniciei o computador.

![Instalacao Ubuntu - ERRO](https://user-images.githubusercontent.com/107226804/201780065-7ea1a93e-0e09-4a9c-a802-05b0a97e6596.PNG)
![Instalacao WSL](https://user-images.githubusercontent.com/107226804/201780070-91cd5628-bb5b-4db1-9ef3-54648e39ebcf.PNG)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @ThierryS99, como vai?

Não sei dizer o que pode ser neste caso, faz um bom tempo que migrei para o Windows 11 e não tive este problema.

Alguns itens que eu olharia:
- Hyper-V 
- Reinstalar o WSL
- Versão do Ubuntu que está instalando (Tentar outra)

**Comment by ThierryS99** [comment]
Boa noite, @andrebaltieri!

Obrigado pela atenção. Consegui a solução do problema num post no Medium. Colocarei o link abaixo caso outro aluno tenha o mesmo problema.

https://wangyj.medium.com/the-solution-to-wsl-error-the-attempted-operation-is-not-supported-for-the-type-of-object-aa559854d1e3#:~:text=of%20object%20referenced%E2%80%9D-,%E2%80%9CThe%20attempted%20operation%20is%20not%20supported%20for%20the%20type%20of,between%20Proxifier%20and%20your%20WSL.

**Reply by andrebaltieri** [reply]
Obrigado por compartilhar 💜
